.gitignore

```diff
@@ -115,3 +115,9 @@ examples/torch/object_detection/eval/
 
 # NNCF debug
 nncf_debug/
+
+# NNCF examples
+examples/experimental/onnx/mobilenet_v2/mobilenet_v2_quantization/
+examples/experimental/openvino/mobilenet_v2/mobilenet_v2_quantization/
+examples/experimental/tensorflow/mobilenet_v2/mobilenet_v2_quantization/
+examples/experimental/torch/mobilenet_v2/mobilenet_v2_quantization/
```

Comment:
Could you please explain why it was done only for `mobilenet_v2`?

Reply:
Because only these samples are work without any additional actions.

Reply:
What do you mean by "additional actions"? How does that relate to the `.gitignore` file?